src/client/kube_store.test.ts

```diff
@@ -54,3 +56,24 @@ test('Test NamespacedResource setResource', () => {
   store.setResource(migResource, planName, examplePlan);
   expect(_.isEqual(expected, store.db)).toBe(true);
 });
+
+test('Test NamespacedResource patchResource', () => {
+  const migResource = new MigResource(MigResourceKind.MigCluster, testNs);
+  const store = new KubeStore('_host');
+  const patch = {'test':'foo'};
+
+  const expected = {
+
+      namespace: testNs,
+        _gvk:
+          { group: 'migration.openshift.io',
+            version: 'v1alpha1',
+            kindPlural: 'migclusters' },
+          test: 'foo',
```

Comment:
This doesn't line up to what we'd normally see.

We'd expect the data to be structured under a 'spec': {}
So would expect something more like
{ 'spec': {'test': 'foo'}}

Reply:
I've added { 'spec': {patch: {'test': 'foo'}}}